docs: add script profiles how to guide (#40)
canonical/landscape-documentation: docs/how-to-guides/web-portal/web-portal-24-04-or-later/use-script-profiles.md

(how-to-web-portal-use-script-profiles)=
# How to use script profiles

Script profiles can be used to execute scripts based on a trigger.

> > **Note**: This feature is only available in **Landscape 25.04** and later.

Reviewer: ````suggestion
```{note}
 This feature is only available in **Landscape 25.04** and later.
```
````

This gives us the "native" notes used by our docs like here: https://documentation.ubuntu.com/landscape/how-to-guides/landscape-installation-and-set-up/install-in-a-lxd-container/#how-to-install-and-configure-landscape-server-in-a-single-lxd-container

## Creating a script profile

From the web portal:

1. Go to **Scripts**.
2. Click on **Profiles**.
3. Click on **Add profile**.

In the script profile creation form, fill out the following fields:

- **Name**: Name of the profile.
- **Script**: Name of a corresponding script (the same access group will be assigned to the profile).
- **Run as user**: The username to execute the script as on the client.
- **Time limit**: The time, in seconds, after which the script is considered defunct.
- **Trigger**:
    - **Post Enrollment**: Triggers after a computer is enrolled in an account.

>     - **On a date**: A point in time the script profile should execute.

Reviewer: ```suggestion
    - **On a date**: A UTC time at which the script profile should execute.
```
